protx-client/src/components/ProTx/components/charts/MainChart.module.scss

```diff
@@ -7,3 +7,18 @@
 .demographics-plot {
   max-height: 55vh;
 }
+.main-chart {
+  min-height: 3vh;
+  margin: 0.5vh 1vw;
+}
+.main-chart-title {
+  display: flex;
+  border-bottom: 1px solid var(--color-gray-very-dark);
+  font-size: calc(var(--font-size-normal) * 1.3);
+  font-weight: var(--font-weight-heavy);
```

Comment:
maybr move into `.main-chart-title-text`?

Reply:
Tried moving this suggestion but it broke the styling needed for this task